src/fios.h

```diff
@@ -26,7 +25,7 @@ enum SaveLoadInvalidateWindowData {
 	SLIWD_FILTER_CHANGES,        ///< The filename filter has changed (via the editbox)
 };
 
-typedef SmallMap<uint, CompanyProperties *> CompanyPropertiesMap;
+using CompanyPropertiesMap = std::map<uint, CompanyProperties *>;
```

Comment:
Rabbit hole-ish: `std::unique_ptr`?

Reply:
Good point, I did it for the font map one already.